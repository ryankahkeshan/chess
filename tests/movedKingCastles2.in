game human human
move d2 d4
move d7 d5
move b1 c4
move b1 c3
move b8 c6
move c1 f4
move c8 f5
move d1 d2
move d8 d7
move e1 d1
move e8 d8
move d1 e1
move d8 e9
move d8 e8
move e1 c1
move e1 d1
move e8 c8
resign
