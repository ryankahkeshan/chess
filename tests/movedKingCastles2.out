8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ P _ _
3 _ _ _ _ 
2 PPP_PPPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 ppp pppp
6  _ _ _ _
5 _ _p_ _ 
4  _ P _ _
3 _ _ _ _ 
2 PPP_PPPP
1 RNBQKBNR

  abcdefgh

Invalid move
8 rnbqkbnr
7 ppp pppp
6  _ _ _ _
5 _ _p_ _ 
4  _ P _ _
3 _ N _ _ 
2 PPP_PPPP
1 R BQKBNR

  abcdefgh

8 r_bqkbnr
7 ppp pppp
6  _n_ _ _
5 _ _p_ _ 
4  _ P _ _
3 _ N _ _ 
2 PPP_PPPP
1 R BQKBNR

  abcdefgh

8 r_bqkbnr
7 ppp pppp
6  _n_ _ _
5 _ _p_ _ 
4  _ P B _
3 _ N _ _ 
2 PPP_PPPP
1 R _QKBNR

  abcdefgh

8 r_ qkbnr
7 ppp pppp
6  _n_ _ _
5 _ _p_b_ 
4  _ P B _
3 _ N _ _ 
2 PPP_PPPP
1 R _QKBNR

  abcdefgh

8 r_ qkbnr
7 ppp pppp
6  _n_ _ _
5 _ _p_b_ 
4  _ P B _
3 _ N _ _ 
2 PPPQPPPP
1 R _ KBNR

  abcdefgh

8 r_ _kbnr
7 pppqpppp
6  _n_ _ _
5 _ _p_b_ 
4  _ P B _
3 _ N _ _ 
2 PPPQPPPP
1 R _ KBNR

  abcdefgh

8 r_ _kbnr
7 pppqpppp
6  _n_ _ _
5 _ _p_b_ 
4  _ P B _
3 _ N _ _ 
2 PPPQPPPP
1 R _K_BNR

  abcdefgh

8 r_ k bnr
7 pppqpppp
6  _n_ _ _
5 _ _p_b_ 
4  _ P B _
3 _ N _ _ 
2 PPPQPPPP
1 R _K_BNR

  abcdefgh

8 r_ k bnr
7 pppqpppp
6  _n_ _ _
5 _ _p_b_ 
4  _ P B _
3 _ N _ _ 
2 PPPQPPPP
1 R _ KBNR

  abcdefgh

Invalid squares
8 r_ _kbnr
7 pppqpppp
6  _n_ _ _
5 _ _p_b_ 
4  _ P B _
3 _ N _ _ 
2 PPPQPPPP
1 R _ KBNR

  abcdefgh

Invalid move
8 r_ _kbnr
7 pppqpppp
6  _n_ _ _
5 _ _p_b_ 
4  _ P B _
3 _ N _ _ 
2 PPPQPPPP
1 R _K_BNR

  abcdefgh

Invalid move
White wins!
Final Score:
White: 1
Black: 0
